submissions/idowuMariam/html-forms/product-page.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
```

Comment:
Hello @Mariamidowu 

You don't have to create a product page in your `html-forms` folder. You already have this file in your `HTML` folder. Kindly link the pages on those already existing file. Like I said. You don't have to create a new file